sdk/communication/communication-tests-template.yml

```diff
@@ -31,3 +31,11 @@ stages:
             failIfCoverageEmpty: true
       EnvVars:
         SKIP_LIVE_TEST: TRUE
+      TestStepMavenInputs:
+        options: '$(DefaultOptions) -Dmaven.javadoc.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -DtrimStackTrace=false -pl $(ProjectList) -Djacoco.skip=$[not(startsWith(System.StageName, ''Public''))]'
```

Comment:
I don't think that approach is going to work here. You might be able to define a variable with a value like that and then use it here but I don't think you can use the runtime `$[...]` expressions inline like this.

Reply:
Yeah. Just trying to limit changes in files in sdk\communication folder. :-)